controller/customdomain_controller_test.go

```diff
@@ -894,6 +902,54 @@ func TestCustomDomainController(t *testing.T) {
 	if err == nil {
 		t.Fatalf("ingresscontroller %s was not deleted!", instanceName)
 	}
+
+	os.Setenv("CLUSTER_VERSION", "4.13.4")
+	defer os.Setenv("CLUSTER_VERSION", "")
```

Comment:
```suggestion
	t.Setenv("CLUSTER_VERSION", "4.13.4")
```
t.Setenv lets you set environment variables for the duration of a test, but with the removal of the environment variable this is probably more of a fun fact